cypress/integration/console/applications/api-keys/api-keys.spec.js

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+describe('Application API keys', () => {
```

Comment:
Since you are on it, please add tests for gateways and organization api-keys. You can use [this PR](https://github.com/TheThingsNetwork/lorawan-stack/pull/3443) as an example on how to structure the tests.